### ISSUE TITLE
[Screen Readers - LxP Assist]: 'Register' button is not associated with their respective labels present inside 'LxP Assist' dialog.

### ISSUE THREAD

**Issue by pgopinadhu** [microsoft/BotFramework-WebChat#4825]
### **PWD Impact:**
Users who rely on screen readers are getting impacted, as 'Register' button is not associated with their respective labels present inside 'LxP Assist' dialog. This will impact user's desired task and will lead to disorientation.

### **Pre-Requisite:**
Enable NVDA/JAWS

### **Repro Steps:**
Step 1: Open the above URL in Edge Chromium/Chrome Browser.
Step 2: 'Enterprise Skills Initiative' page will appear.> Navigate to the 'Click to open Copilot' button and click on it. > 'LXP Assist' flyout will appear >Navigate to the 'Register' control using tab key.
Step 3: Observe whether 'Register' button is not associated with their respective labels present inside 'LxP Assist' dialog or not.

### **Actual Result:**
'Register' button is not associated with their respective labels present inside 'LxP Assist' dialog.
Screen Reader Announces: Register toggle button not pressed Register

**Observation:**
There are multiple 'Register' controls are present on the page and screen reader is announcing same information on all the 'Register' control.

### **Expected Result:**
'Register' button should be associated with their respective labels present inside 'LxP Assist' dialog.
Screen reader should announce: Microsoft Azure Virtual Training Day: Implementing Hybrid Infrastructure Register link opens in a new tab

### **MAS Reference:** 
[MAS 3.3.2 – Labels or Instructions](https://nam06.safelinks.protection.outlook.com/ap/w-59584e83/?url=https:%2f%2fmicrosoft.sharepoint.com%2f:w:%2fr%2fsites%2faccessibility%2f_layouts%2f15%2fDoc.aspx?sourcedoc%3D%257B3752B546-30D8-44D7-8221-6FC32EEB3AC8%257D%26file%3DMAS%25203.3.2%2520%25E2%2580%2593%2520Labels%2520or%2520Instructions.docx%26wdLOR%3Dc36957864-4B45-43F1-AB1E-E2E3A9834EFC%26action%3Ddefault%26mobileredirect%3Dtrue&data=05%7c01%7cv-venkatab%40microsoft.com%7c152811a2bec143d2588808dad9b5b040%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638061671699303779%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=FnZ8bW%2fmU4IGRomcx2dWTitGBuK25fKrHbuVdmZIAUw%3D&reserved=0)

### **WCAG Reference:**
[https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fwww.w3.org%2fWAI%2fWCAG21%2fUnderstanding%2flabels-or-instructions.html&data=05%7c01%7cv-venkatab%40microsoft.com%7c152811a2bec143d2588808dad9b5b040%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638061671699303779%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=QToQIVVSI7%2f3DxIHjkoY4X%2fOygw8Wwjj7gkVLqq3UMo%3D&reserved=0)
### **Test Environment:**
OS: Windows 11 22H2 (OS Build 22621.1992)
Browser: Microsoft Edge Version (114.0.1823.82), Chrome (114.0.5735.199)
Screen Reader: NVDA (2023.1), JAWS (J2023.2303.144.400)
URL: [https://ui.int.esi.microsoft.com/](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fui.int.esi.microsoft.com%2f&data=05%7c01%7cv-rmounika%40microsoft.com%7c4ed10265b9a246cbbba108db84c198c6%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638249739385137411%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=NWSDitB8DvvJww0%2fMAZw1XoxXQrpuW14b89LtmxoRpU%3D&reserved=0)

**Comment by pgopinadhu** [comment]
![image](https://github.com/microsoft/BotFramework-WebChat/assets/120019572/f6da80ba-6281-4dd1-84c2-de21fc76665b)

**Comment by pgopinadhu** [comment]
#HCL; #Accessibility; #Benchmark; #A11yMAS; #A11ySev2; #MAS3.3.2; #Win11-Edge(Chromium); #FeatureBotframeworkwebchat-Apr23; #WWL:Bug1654480; #WWL;

**Comment by compulim** [comment]
Same as #4826. This is Adaptive Cards issue. Not Web Chat.
